src/js/components/DropButton/__tests__/__snapshots__/DropButton-test.js.snap

```diff
@@ -47,17 +42,13 @@ exports[`DropButton close by clicking outside 1`] = `
   border-radius: 18px;
   color: #444444;
   padding: 4px 22px;
+  font-size: 18px;
```

Comment:
It looks like we may have lost the line-height for the label. If so, that seems like it might cause some alignment issues.

Reply:
let me look into this.

Reply:
Thanks for fixing. This is a good example of snapshot tests working, if we take the time to review the snapshot deltas.

Reply:
agreed.